packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -48,17 +50,30 @@ export const FinalFormSelect = <T,>({
             return String(option);
         }
     },
-
+    reload,
     noOptionsLabel = (
         <Typography variant="body2">
             <FormattedMessage id="finalFormSelect.noOptions" defaultMessage="No options." />
         </Typography>
     ),
-    errorLabel = (
-        <Typography variant="body2">
-            <FormattedMessage id="finalFormSelect.error" defaultMessage="Error loading options." />
-        </Typography>
+    renderErrorLabel = (reloadFunction) => (
+        <MenuItemContainerContent>
+            <Typography variant="body2">
+                <FormattedMessage id="finalFormSelect.error" defaultMessage="Error loading options." />
+            </Typography>
+            {reloadFunction && (
+                <IconButton
+                    onClick={(event) => {
+                        event.stopPropagation();
+                        reloadFunction?.();
+                    }}
+                >
+                    <Reload fontSize="small" />
+                </IconButton>
+            )}
```

Comment:
Could we instead use MUI's [ListItemText](https://mui.com/material-ui/api/list-item-text/) and [ListItemSecondaryAction](https://mui.com/material-ui/api/list-item-secondary-action/) for this?